### PR TITLE
config: drop commented config lines

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

> /kind api-change
/kind bug
> /kind ci
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
> /kind other

#### What this PR does / why we need it:
now that we only print fields that are different from the default,
we can uncomment the config template fields that we previously didn't want to mess with.

Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Some fields in the crio configuration related to storage and images were commented out by default (when running `crio config`). They have been uncommented, and will be applied (if they're different from the default value).
```

### CODE REVIEW

**Comment by haircommander** [comment]
want a +1 from @mrunalp because he didn't want this before (but now we have a non-default-only crio.conf!)

File: pkg/config/template.go [hunk 1]
```diff
@@ -1050,7 +1050,8 @@ signature_policy = "{{ .SignaturePolicyPath }}"
 const templateStringCrioImageInsecureRegistries = `# List of registries to skip TLS verification for pulling images. Please
 # consider configuring the registries via /etc/containers/registries.conf before
 # changing them here.
-#insecure_registries = "{{ .InsecureRegistries }}"
+#insecure_registries = [
```
Comment (fidencio) [inline pkg/config/template.go:1053]: nit: as you're already changing the others adding a space between "#" and the first letter, please, do the same here.

**Comment by TomSweeneyRedHat** [comment]
Did something go askew in your rebasing?  I'm only seeing one file now and it's not intuitive how it ties in to the description by itself....

**Comment by haircommander** [comment]
> Did something go askew in your rebasing? I'm only seeing one file now and it's not intuitive how it ties in to the description by itself....

sorry, this is still wip I am debugging why a test is failing

**Review by mrunalp — APPROVED** [review]

**Comment by haircommander** [comment]
/override ci/prow/e2e-gcp
agnostic passed

**Pull request merged**